includes/class-core.php

```diff
@@ -200,135 +200,142 @@ public function get_module( string $name ) {
-
-		// Include custom fields before and after flexible content field.
-		$field_group_fields =
-			array_merge( $fields_before_flexible_content, [
-				[
-					'type'         => 'flexible_content',
-					'key'          => 'hogan_' . $name . '_modules_key', // i.e. hogan_default_modules_key.
-					'name'         => 'hogan_' . $name . '_modules_name',
-					'button_label' => esc_html__( 'Add module', 'hogan-core' ),
-					'layouts'      => $field_group_layouts,
-				],
-			], $fields_after_flexible_content );
-
-		$location = array_merge( $this->get_post_type_location( $name ), $location );
+		// Set field group key based on name, i.e. hogan_default.
+		$args['key'] = 'hogan_' . $args['name'];
 
-		acf_add_local_field_group(
+		// Set field group fields from modules.
+		$args['fields'] = array_merge( $args['fields_before_flexible_content'], [
 			[
-				'key'            => 'hogan_' . $name, // i.e. hogan_default.
-				'title'          => $label,
-				'fields'         => $field_group_fields,
-				'location'       => apply_filters( 'hogan/field_group/' . $name . '/location', $location ),
-				'hide_on_screen' => apply_filters( 'hogan/field_group/' . $name . '/hide_on_screen', $hide_on_screen ),
-			]
-		);
+				'type'         => 'flexible_content',
+				'key'          => 'hogan_' . $args['name'] . '_modules_key',
+				'name'         => 'hogan_' . $args['name'] . '_modules_name',
+				'button_label' => __( 'Add module', 'hogan-core' ),
+				'layouts'      => $field_group_layouts,
+			],
+		], $args['fields_after_flexible_content'] );
+
+		// Add to core field groups.
+		$this->_field_groups[] = $args['name'];
+
+		// Create ACF field group.
+		acf_add_local_field_group( $args );
 	}
 
 	/**
-	 * Add location rule to post types.
+	 * Register default field group for all modules.
 	 *
-	 * @param string $name Field group name.
-	 *
-	 * @return array New location rules.
+	 * @return void
 	 */
-	private function get_post_type_location( string $name ) : array {
-		$supports_hogan_field_group = apply_filters( 'hogan/supported_post_types', [ 'page' ], $name );
+	public function register_default_field_group() {
 
-		$location = [];
+		$location   = [];
+		$post_types = apply_filters( 'hogan/field_group/default/supported_post_types', [ 'page' ] );
 
-		if ( ! empty( $supports_hogan_field_group ) && is_array( $supports_hogan_field_group ) ) {
-			foreach ( $supports_hogan_field_group as $post_type ) {
+		// Include the deprecated global filter, to be removed at some point.
+		$post_types = apply_filters_deprecated( 'hogan/supported_post_types', [ $post_types ], '1.3.0', 'hogan/field_group/default/supported_post_types' );
+
+		if ( is_array( $post_types ) && ! empty( $post_types ) ) {
+			// Add default location parameter for each supported post type.
+			foreach ( $post_types as $post_type ) {
 				$location[] = [
 					[
-						'param'    => 'post_type',
-						'operator' => '==',
-						'value'    => $post_type,
+						'param' => 'post_type',
```

Comment:
Missing `'operator' => '==',`

Reply:
`==` is the default acf operator and is not needed here.
<img width="286" alt="screen shot 2018-04-24 at 13 27 26" src="https://user-images.githubusercontent.com/16900754/39184371-6a42d204-47c3-11e8-8858-675e75373321.png">

Reply:
Fixed.